src/Symfony/Component/Security/CHANGELOG.md

```diff
@@ -1,6 +1,11 @@
 CHANGELOG
 =========
 
+4.3.0
+-----
+
+* [BC BREAK] Removed `\Serializable` interface from `AbstractToken` and `AuthenticationException`.
```

Comment:
Made the `serialize()` and `unserialize()` of `AbstractToken` and `AuthenticationException` final, use `getState()`/`setState()` instead